authdelegate/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash -e
+
+source /etc/profile.d/gvm.sh
+
+if [ "$1" = "run-server" ]; then
```

Comment:
Yes; when running `./go run_container nginx /bin/bash` or some such to set up the environment but drop into a shell to poke around without running the service. Also, in #8, I found out how to use the same `hmacproxy` image to run an auth server and a signing proxy in two different containers, depending on if it's started with `run-server` or `run-proxy`.

I picked up the idea from https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#entrypoint.